plugin/othttp/handler_test.go

```diff
@@ -100,3 +100,41 @@ func TestBasicFilter(t *testing.T) {
 		t.Fatalf("got %q, expected %q", got, expected)
 	}
 }
+
+func TestSpanFormatter(t *testing.T) {
+	rr := httptest.NewRecorder()
+
+	var id uint64
+	var spanName string
+	tracer := mocktrace.MockTracer{
+		StartSpanID: &id,
+		OnSpanStarted: func(span *mocktrace.MockSpan) {
+			spanName = span.Name
+		},
+	}
+
+	h := NewHandler(
+		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if _, err := io.WriteString(w, "hello world"); err != nil {
+				t.Fatal(err)
+			}
+		}),
+		"",
+		WithTracer(&tracer),
+		WithSpanNameFormatter(func(_ string, r *http.Request) string {
+			return r.URL.Path
+		}),
+	)
+
+	r, err := http.NewRequest(http.MethodGet, "http://localhost/hello", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	h.ServeHTTP(rr, r)
+	if got, expected := rr.Result().StatusCode, http.StatusOK; got != expected {
+		t.Fatalf("got %d, expected %d", got, expected)
+	}
+	if got, expected := spanName, "/hello"; got != expected {
+		t.Fatalf("got %q, expected %q", got, expected)
+	}
+}
```

Comment:
Please add a test for default span formatter.

Reply:
@rghetia done :)